rustls/src/client/client_conn.rs

```diff
@@ -244,6 +239,68 @@ impl ClientConfig {
     }
 }
 
+/// Configuration for how/when a client is allowed to resume a previous session.
+#[derive(Clone)]
+pub struct Resumption {
+    /// How we store session data or tickets.
+    pub(super) store: Arc<dyn ClientSessionStore>,
+
+    /// The default is true.
```

Comment:
I think we'll want to change the wording here. Default is `in_memory_sessions(256)` which implies `Some(Tls12Resumption::SessionIdOrTickets)` as the default for `tls12_resumption`

Reply:
Makes sense.

Reply:
Reworded in f906c29c399f3a5347ed6118607677929edf78b9